apluslms_roman/cli.py

```diff
@@ -139,7 +139,11 @@ def create_parser(version=__version__,
         action='version',
         version="%%(prog)s %s" % (version,),
         help=_("print a version info and exit"))
-
+    parser.add_argument('-s', '--steps',
```

Comment:
These should be set only for the `build` sub action, so define them for `build`parser here: https://github.com/apluslms/roman/blob/5d77d5391bc6b657aa25b98892aff08673ba153b/apluslms_roman/cli.py#L199

Reply:
do you mean both --steps and --buildsteps? why should --steps be added to build?

Reply:
if I move --buildsteps to build I can't get the values from context.parser.parse_args(). where can I find the buildsteps? (at least I think it was this that broke it)

Reply:
I fixed how base parser defaults are set when no subcommand is given. So, this should work nicely now. You need to rebase your changes on top of current milestone0.3